oat-sa/extension-tao-itemqti
# Feature/ms 772/prevent scrolling text entry

Ticket: https://oat-sa.atlassian.net/browse/MS-772

### What's changed?

- Style of the inline text entry interaction is changed according to the new design

### How to test?

- Create an item with inline text entry and publish the delivery (enable manual scoring)
- Take the delivery as a TT
- Login to MS as a scorer and open the response
- The inline text entry interaction in MS should work according to the ACs defined in the ticket
